### PR TITLE
Datatables

### PR DESCRIPTION
**Pull request by nandvadera1**

### CODE REVIEW

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    public function create()
+    {
+        $type = UserType::pluck('type', 'id');
+        return view('users.create', compact('type'));
+    }
+
+    public function store(Request $request)
+    {
+//        dd($request->input('user_type_id'));
```
Comment (gadhiamitul41) [inline app/Http/Controllers/UserController.php]: Any debug code should not be part of the code. Please clean it

File: config/adminlte.php [hunk 2]
```diff
@@ -290,99 +290,104 @@
...
-            'text'       => 'warning',
-            'icon_color' => 'yellow',
-            'url'        => '#',
-        ],
-        [
-            'text'       => 'information',
-            'icon_color' => 'cyan',
-            'url'        => '#',
-        ],
+          [
```
Comment (gadhiamitul41) [inline config/adminlte.php:293]: remove all the commented code to keep the master branch clean.

**Review by nandvadera1 — COMMENTED** [review]
nand

**Comment by urvish2442** [comment]
Hello

File: config/adminlte.php (hunk 2 above)
Comment (gadhiamitul41) [reply config/adminlte.php:293]: @nandvadera1 : Commented code means any PHP Code that is left over from the development or testing has to be removed not every comment.

File: resources/views/users/create.blade.php [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            @include('users._form')
+            <div class="card-footer">
+                {!! Form::submit('Submit', ['class' => 'btn btn-primary']) !!}
+            </div>
+            {!! Form::close() !!}
+        </div>
+    </div>
+@stop
+@section('css')
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap.min.css">
```
Comment (gadhiamitul41) [inline resources/views/users/create.blade.php]: @nandvadera1 : Why do we have this css in every page we have created? I think we don't need this

File: resources/views/users/create.blade.php [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    <style>
+        .error {
+            color: red;
+            font-size: small;
+        }
+    </style>
+@stop
+
+@section('js')
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
```
Comment (gadhiamitul41) [inline resources/views/users/create.blade.php]: @nandvadera1 :  I think jquery is already loaded by the main template

File: resources/views/users/create.blade.php (hunk 3 above)
Comment (nandvadera1) [reply resources/views/users/create.blade.php]: yes removed.

File: resources/views/users/create.blade.php (hunk 4 above)
Comment (nandvadera1) [reply resources/views/users/create.blade.php]: yes removed.

**Pull request merged**